sdk/Trace/ZipkinExporter.php

```diff
@@ -53,15 +54,11 @@ public function export(iterable $spans) : int
 
         try {
             $json = json_encode($convertedSpans);
-            $contextOptions = [
-                'http' => [
-                    'method' => 'POST',
-                    'header' => 'Content-Type: application/json',
-                    'content' => $json,
-                ],
-            ];
-            $context = stream_context_create($contextOptions);
-            file_get_contents($this->getEndpointUrl(), false, $context);
+            $url = $this->getEndpointUrl();
+            $client = new \GuzzleHttp\Client();
+            $headers = ['content-type' => 'application/json'];
+            $request = new Request('POST', $url, $headers, $json);
+            $response = $client->send($request, ['timeout' => 2]);
```

Comment:
This is the default timeout for guzzle; it's probably much too long but we can change it later if we deem prudent.

Reply:
The [SDK/Tracing spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-tracing.md) says this for a parameter to batching processors:

> exporterTimeoutMillis - how long the export can run before it is cancelled. The default value is 30000.

That default looks insanely high in traditional PHP SAPIs, but the point is that it's configurable, I think?

Reply:
Yeah; I can think of a couple ways we could change this.  30000ms (30s) does feel insanely long, but we should follow the spec I guess and change it if we deem important later.  Updating.

Reply:
I would suggest using some kind of dependency injection approach. I would add the possibility to inject the http-client in the Exporter, and only if no client is injected, then I would instantiate a default http-client. I would use PSR18 http-client interface.